web-local/src/lib/components/workspace/core/WorkspaceHeaderStatusSpinner.svelte

```diff
@@ -27,10 +27,10 @@
   $: applicationStatusTooltip = applicationStatusTooltipMap[applicationStatus];
 </script>
 
-<div>
+<div class="mr-2">
```

Comment:
I think we should probably keep this spacing in the parent component rather than here. Otherwise this affects reuse.